PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+## Why?
+
+( Please include a rationale for the changes and the related issue, providing any relevant motivation and context. )
+
+( Please include a link to the Linear ticket(s). If there is no ticket associated, explain why. )
+( e.g. [ARCH-66 : chore: Create a standard PR Template](https://linear.app/archipelago-ai/issue/ARCH-66/chore-create-a-standard-pr-template) )
+
+## This PR
+
+( Please summarize the changes in this PR, using plain English wherever possible. )
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
```

Comment:
This is what our Conventional Commits PR naming convention is for. This is redundant. If anything, we should modify CI to ensure PR titles conform to guidelines through an action like https://github.com/marketplace/semantic-prs